src/main/java/seedu/contax/model/onboarding/OnboardingStep.java

```diff
@@ -132,6 +147,6 @@ public boolean equals(Object other) {
     public int hashCode() {
         return Objects.hash(eventType, displayMessage, overlayOption,
                 messageHeight, messageWidth, highlightOption,
-                positionOption, person, command, getPersonOperation());
+                positionOption, person, command, getOperationId(), isCommandCustom);
```

Comment:
Why did you mix in a getter method in the middle of it.

Reply:
Fixed